.github/workflows/distcheck.yaml

```diff
@@ -56,6 +56,12 @@ jobs:
           - latest
           - 1.25.0
```

Comment:
We have an internal action to get more coverage with the latest _dev_ installer as well. Should extend this at that point.

Reply:
but i'm not really convinced that helps for what are effectively build only tests